### ISSUE TITLE
active logic integration?

### ISSUE THREAD

**Issue by slimshader** [active-logic/xgoap#58]
Hey, is mentioned integration with Active Logic still coming?

**Comment by eelstork** [comment]
Can't think why not. Do you have a concrete case/angle to help us think about this? Otherwise I'll just do a small spike/test to identify pain points.

**Comment by slimshader** [comment]
I mean I was just asking about the "[COMING SOON] integration with the BT Framework of Awesome, Active Logic 🚀" part but really was also wondering if there are updates coming for xgoap or active logic

**Comment by eelstork** [comment]
Yes, both libraries will receive more attention although I had to go on a little hiatus while I was busy with another project. Guessing we'll see more changes starting from May.

Please note:
- Because Active Logic has received intense focus already, I will mainly focus on fixing bugs when found, and encouraging others (yes, there's a few handy features that I do want personally too) to try the library. Judging by the disproportionate interest around XGOAP, not sure... ...maybe reformulate the free version or rename the lib... (?)
- For XGOAP, the situation is quite different as I consider the library is a work in progress, and I need a high level (no code) integration later on. So when the dust settled around performance and integration topics, likely headed this way (library core will of course remain available for just happy coding).

In sum my personal commitment is high, which for now (pending meaningful adoption) is enough to motivate more dev.

I recall we have talked before? Did you have any experience we either of these thus far?

**Comment by eelstork** [comment]
If this issue (as per title) is not a duplicate let's leave it open until properly resolved.
